docs/pages/implementation/nuts-auth.rst

```diff
@@ -0,0 +1,225 @@
+
+.. code-block::
+
+    {
+      "bearer_token": "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsIng1YyI6WyJNSUlDZ0RDQ0FXaWdBd0lCQWdJUkFLT21reVp5Z2lBRXFMeUt1T3JnKzNjd0RRWUpLb1pJaHZjTkFRRUxCUUF3TmpFTE1Ba0dBMVVFQmhNQ1Rrd3hFVEFQQmdOVkJBb1RDRlpsYm1SdmNpQkNNUlF3RWdZRFZRUURFd3RXWlc1a2IzSWdRaUJEUVRBZUZ3MHlNREV5TURFeE1EUXdOVFphRncweU1ERXlNRFV4TURRd05UWmFNRGt4Q3pBSkJnTlZCQVlUQWs1TU1SRXdEd1lEVlFRS0V3aFdaVzVrYjNJZ1FqRVhNQlVHQTFVRUF4TU9WbVZ1Wkc5eUlFSWdiMkYxZEdnd1dUQVRCZ2NxaGtqT1BRSUJCZ2dxaGtqT1BRTUJCd05DQUFSOUxvREF4clVTM1NxM2I4akZmakdmOE5IUXR2ODBwWXFobENhaEd0djRjM0w3a1M3c1BmR0I4VjZ5WkNZVURyWXU5ZUdWcEJFQUtBL0MwWnNUWHhsWm8xRXdUekFPQmdOVkhROEJBZjhFQkFNQ0JzQXdJZ1lEVlIwUkJCc3dHYUFYQmdrckJnRUVBWU9zUXdTZ0Nnd0lNREF3TURBd01ESXdHUVlKS3dZQkJBR0RyRU1EQkF3TUNtaGxZV3gwYUdOaGNtVXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBQlNSVTNGKys4VUsvOEthVHBBWGZUOHUxd0t6ZFpEUDB5NTNNNTZET2xqK0JmaWNsd0h0Rlc2VmhvTU5sSXBUU1NCUWkxS0hKZWZnVUlGNStRd0lDbE9jeDNheDNldzJBVVNEVWdURnBiS2pETzF3VjhCWWw4dGdjdjkzc2NNZ0pTM09ZeGZnY1VQZ1AvVEZMK1R2SVNMb3pWcURBOGVTMDUyREdEM0MvbS80QjVKZEVkYU9tSVdTdzk2Z1g2LzYrTHI4TUJHWGp6aEFKa2RXcmEzeUhDS2R4ME4xSVpkUW53SnZacG02ZVRpTkhSbHgyQUkrMTVpVXdNaWdlTlhSazhzNFEybG9vb2FIUWhtcWpXSTU4Z3ZHa1l3YmZ3Yk91b2UrRFdLQ2xWMkJnZjcvanlwRU9JTUhNYS9SV0RzUHhxdy9iT3FRaDBwZ3BwRU82TzFHQytBPSJdfQ.eyJTaWduaW5nQ2VydGlmaWNhdGUiOm51bGwsImF1ZCI6IjRlYjlmNDM4LTc4MWQtNDU5NC04NTMxLTNiODgxOTc1MWM4YyIsImV4cCI6MTYwNjgxOTI2MSwiaWF0IjoxNjA2ODE5MjU2LCJpc3MiOiJ1cm46b2lkOjIuMTYuODQwLjEuMTEzODgzLjIuNC42LjE6QiIsInNjb3BlIjoiIiwic2lkIjpudWxsLCJzdWIiOiJ1cm46b2lkOjIuMTYuODQwLjEuMTEzODgzLjIuNC42LjE6QSIsInVzaSI6ImV5SkFZMjl1ZEdWNGRDSTZXeUpvZEhSd2N6b3ZMM2QzZHk1M015NXZjbWN2TWpBeE9DOWpjbVZrWlc1MGFXRnNjeTkyTVNKZExDSndjbTl2WmlJNmV5SkNhWEowYUdSaGRHVWlPaUl4T1Rnd0xUQXhMVEF4SWl3aVEyOXVkSEpoWTNRaU9pSkZUanBRY21GamRHbDBhVzl1WlhKTWIyZHBianAyTXlCSklHaGxjbVZpZVNCa1pXTnNZWEpsSUhSdklHRmpkQ0J2YmlCaVpXaGhiR1lnYjJZZ1QzSm5JRUl1SUZSb2FYTWdaR1ZqYkdGeVlYUnBiMjRnYVhNZ2RtRnNhV1FnWm5KdmJTQmthVzV6WkdGbkxDQXhJR1JsWTJWdFltVnlJREl3TWpBZ01URTZOREE2TlRZZ2RXNTBhV3dnWkdsdWMyUmhaeXdnTVNCa1pXTmxiV0psY2lBeU1ESXdJREV5T2pRd09qVTJMaUlzSWtWdFlXbHNJam9pZEdWemRHVnlRR1Y0WVcxd2JHVXVZMjl0SWl3aVNXNXBkR2xoYkhNaU9pSkpJaXdpVEdGemRHNWhiV1VpT2lKVVpYTjBaWElpTENKVWVYQmxJam9pVG05VGFXZHVZWFIxY21VaWZTd2lkSGx3WlNJNld5SldaWEpwWm1saFlteGxVSEpsYzJWdWRHRjBhVzl1SWl3aVJIVnRiWGxXWlhKcFptbGhZbXhsVUhKbGMyVnVkR0YwYVc5dUlsMTlDZz09In0.b-HfO25rqmMusqFGxXiZ645bKvgZTfJL11er_Fm1Svc7gcubhGkWSOR9zuNdW_nYHwTDQI4z7pVkUqgdEd-BhA"
+    }
+
+This bearer token can be used to get an access token at the custodian side.
+
+Access token
+************
+
+The access token is issued by the authorization server at the custodian side. According to spec it may be anything as long as it's BASE64 encoded and it can fit in a HTTP header. The authorization server embedded in the Nuts OS implementation uses JWTs as access tokens.
+
+To get an access token, perform the following steps:
+
+- find the authorization server endpoint
+- retrieve the access token
+- use the access token
```

Comment:
'use the access token' is probably not a step but rather the goal of the 'to get an access token' process